src/client/e2e/spot-tile.spec.ts

```diff
@@ -40,53 +40,50 @@ test.describe("Spot Tile", () => {
       expect(tradeId).toBe(blotterTradeID)
     })
 
-    test("When I buy USD/JPY then an Execution screen appears until transaction is performed and tile displays green confirmation screen", async () => {
+    test("When I buy USD/JPY then a tile displays in green with confirmation message", async () => {
       await tilePage.locator("[data-testid='Buy-USDJPY']").click()
 
-      const executingSpinner = tilePage.getByText(/Executing/)
-      expect(executingSpinner).not.toBe(null)
-
       const greenConfirmation = tilePage
         .locator("div[role='dialog']")
         .getByText(/You bought/)
-      expect(greenConfirmation).not.toBe(null)
+      await expect(greenConfirmation).toBeVisible()
     })
   })
 
   test.describe("Rejected purchase confirmation", () => {
-    test("When I buy GBP/JPY then an execution screen appears and the tile displays are red with message Trade was rejected", async () => {
+    test("When I buy GBP/JPY then a tile displays in red with message 'Trade was rejected'", async () => {
       await tilePage.locator("[data-testid='Buy-GBPJPY']").click()
 
-      const executingSpinner = tilePage.getByText(/Executing/)
-      expect(executingSpinner).not.toBe(null)
-
-      const greenConfirmation = tilePage
+      const redConfirmation = tilePage
         .locator("div[role='dialog']")
         .getByText(/Your trade has been rejected/)
-      expect(greenConfirmation).not.toBe(null)
+      await expect(redConfirmation).toBeVisible()
     })
   })
 
   test.describe("Timed out transaction", () => {
     test.setTimeout(60000)
-    test("When I sell EUR/JPY then a timed out message appears with message displays 'Trade taking longer than expected'", async () => {
+    test("When I sell EUR/JPY then an execution animation appears until a timed out tile displays in orange with message 'Trade taking longer than expected'", async () => {
       await tilePage.locator("[data-testid='Sell-EURJPY']").click()
 
-      const greenConfirmation = await tilePage
+      const executingSpinner = tilePage.getByText(/Executing/)
+      await expect(executingSpinner).toBeVisible()
+
+      const orangeconfirmation = await tilePage
```

Comment:
Very minor typo

Reply:
@SHolleworth agree thanks, I've corrected it

Reply:
@SHolleworth can you review again for approval ? thanks

Reply:
Sorry, feel free to ping me on Slack for something like this if you need